### PR TITLE
Milestone 1: Review previous collaboration trial project

### PR DESCRIPTION
**Pull request by mavericks-db**
## Technical Documentation

This branch includes:
-add index.html
-add style.css

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+html,
```
Comment (mavericks-db) [inline css/style.css:1]: Please apply linter checks to correct proper formatting. You can find the linter instructions here: https://github.com/microverseinc/linters-config

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+body {
+    min-width: 290px;
+    background-color: white;
+    font-family: Arial, sans-serif;
+    line-height: 2em;
+}
+#navbar {
+    position:fixed;
+    text-align: left;
+    margin : 0px auto 0px auto;
```
Comment (mavericks-db) [inline css/style.css:11]: You can use `margin: 0 auto` for shorthand.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,555 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (mavericks-db) [inline index.html:551]: You can delete comments if unnecessary on your HTML file. Too many comments will make your code messy.

**Review by mavericks-db — COMMENTED** [review]
## STATUS: CHANGES REQUIRED ♻️
## Hi there 👋, 

Good job so far!
However, there are some issues that you still need to work on.
Please have a look at the comments below.

## Required Changes ♻️

[ ] Use linters for format check, there is a link provided in the comments.
[ ] Customize README file, please see the link below.
[ ] Check the comments under the review.

Cheers and Happy Coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**Review by MeqdamAlqudah — APPROVED** [review]
# Hi @mavericks-db , 

### Status: Approved ✔️

:+1:  You have implemented all the suggested changes made by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**